tests/jballerina-unit-test/src/test/resources/test-src/balo/imports/test-case4/main.bal

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import testOrg/testModule version 1.1.0 as foo;
```

Comment:
Do we have tests where a version is specified but there is no alias? Shall we add one if not?

e.g.,
```suggestion
import testOrg/testModule version 1.1.0;
```

Reply:
Fixed in 20b220c